packages/components/pie-webc-core/src/test/forms/pie-form-manager.spec.ts

```diff
@@ -0,0 +1,198 @@
+import {
+    describe,
+    it,
+    expect,
+    vi,
+} from 'vitest';
+
+import { PieFormManager } from '../../forms/pie-form-manager';
+
+describe('PieFormManager', () => {
+    describe('addForm', () => {
+        it('correctly sets up a submit event listener when a form is added', () => {
+            // Arrange
+            document.body.innerHTML = `
+                <form id="testForm"></form>
+            `;
+
+            const testForm = document.getElementById('testForm') as HTMLFormElement;
+            const manager = new PieFormManager();
+            const addEventListenerSpy = vi.spyOn(testForm, 'addEventListener');
+
+            // Act
+            manager.addForm(testForm);
+
+            // Assert
+            expect(addEventListenerSpy).toHaveBeenCalledWith('submit', expect.any(Function));
```

Comment:
What's triggering the form submission here?

Reply:
There is no form submission, this is checking that `addEventListener` was called for the submit event